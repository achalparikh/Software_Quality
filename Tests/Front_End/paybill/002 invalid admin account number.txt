login
admin
paybill
ACHAL PARIKH
00002
EC
100.00
logout
quit