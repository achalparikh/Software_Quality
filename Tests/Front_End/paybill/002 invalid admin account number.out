Enter user type - "standard" or "admin": 
You are now logged in as an admin 
COMMAND: paybill - enter user: 
Paying bill for "ACHAL PARIKH" - enter account number:
ERROR: Account number "00002" does not exist. Try again
You are now logged out
